evaluation/scripts/service_setup.py

```diff
@@ -0,0 +1,159 @@
+import logging
+import os
+
+import openai
+from azure.core.credentials import AzureKeyCredential
+from azure.identity import AzureDeveloperCliCredential, get_bearer_token_provider
+from azure.search.documents import SearchClient
+from promptflow.core import (
+    AzureOpenAIModelConfiguration,
+    ModelConfiguration,
+    OpenAIModelConfiguration,
+)
+from pyrit.chat_message_normalizer import ChatMessageNop, ChatMessageNormalizer
+from pyrit.prompt_target import (
+    AzureMLChatTarget,
+    AzureOpenAIChatTarget,
+    OpenAIChatTarget,
+    PromptChatTarget,
+)
+
+logger = logging.getLogger("scripts")
+
+def print_all_env_vars():
+    vars = ['OPENAI_HOST', 'OPENAI_GPT_MODEL', 'AZURE_OPENAI_EVAL_DEPLOYMENT', 'AZURE_OPENAI_SERVICE', 'OPENAICOM_KEY', 'OPENAICOM_ORGANIZATION', 'AZURE_SEARCH_SERVICE', 'AZURE_SEARCH_INDEX', 'AZURE_OPENAI_KEY', 'TENANT_ID', 'CLIENT_ID', 'CLIENT_SECRET', 'AZURE_PRINCIPAL_ID']
+    print('Environment Variables:')
+    for var in vars:
+        print(f"{var}: {os.environ.get(var)}")
```

Comment:
In your README you should also provide instructions on how one can find these values. Perhaps they're already set in local azd env vars for instance and you can retrieve them directly from there. It's crucial that those things can be automated. You can describe a manual process that involves finding the corresponding information on the Azure portal, but you should also provide a way to fetch the values following a successful deployment.

Reply:
> In your README you should also provide instructions on how one can find these values. Perhaps they're already set in local azd env vars for instance and you can retrieve them directly from there. It's crucial that those things can be automated. You can describe a manual process that involves finding the corresponding information on the Azure portal, but you should also provide a way to fetch the values following a successful deployment.

I've dealt with all the comments I can so far and modified the readme file as requested. PTAL thanks!